action.yml

```diff
@@ -0,0 +1,45 @@
+---
+name: 'Acquia Database Backup and S3 Sync'
+description: 'Sync existing database backups/dump from Acquia to S3.'
+author: 'Adam Zimmermann'
+inputs:
+  acquia_environment:
+    description: 'Acquia environment name.'
+    default: 'prod'
+    required: true
+  acquia_project:
+    description: 'Acquia project ID.'
+    required: true
+  acquia_private_key:
+    description: 'Acquia Private SSH Key.'
+    required: true
+  acquia_public_key:
+    description: 'Acquia Public SSH Key.'
+    required: true
+  aws_s3_bucket:
```

Comment:
I'm using a test bucket for now to avoid blowing away backups we want. I will need to change the value of this variable in the consuming repo.